src/edu/csus/ecs/pc2/clics/API202306/ProblemService.java

```diff
@@ -0,0 +1,143 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.util.ArrayList;
+
+import javax.inject.Singleton;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Feature;
+import javax.ws.rs.core.FeatureContext;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.SecurityContext;
+import javax.ws.rs.ext.Provider;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.IInternalController;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.core.model.Problem;
+import edu.csus.ecs.pc2.core.util.IJSONTool;
+import edu.csus.ecs.pc2.services.core.JSONUtilities;
+import edu.csus.ecs.pc2.services.eventFeed.WebServer;
+
+/**
+ * WebService to handle problems
+ *
+ * @author John Buck
+ *
+ */
+@Path("/contests/{contestId}/problems")
+@Produces(MediaType.APPLICATION_JSON)
+@Provider
+@Singleton
+public class ProblemService implements Feature {
+
+    private IInternalContest model;
+
+    @SuppressWarnings("unused")
+    private IInternalController controller;
+
+    private JSONTool jsonTool;
```

Comment:
It appears that `jsonTool` is not used anywhere in this class.  (It's interesting that THIS class declares such a thing whereas the other service classes don't...)